### PR TITLE
Dockerfile versions

### PR DESCRIPTION
**Pull request by johnbradley**
WIP: Adding versions/tags to docker images.
Also fixing images that would have changed versions by rebuilding.

### CODE REVIEW

File: samtools/Dockerfile.1.3 [hunk 1]
```diff
@@ -23,4 +23,4 @@ RUN curl -SLo ${DEST_DIR}/samtools-${SAMTOOLS_RELEASE}.tar.bz2 ${SAMTOOLS_URL}/$
   make install && \
   rm -rf ${DEST_DIR}/samtools-${SAMTOOLS_RELEASE}
 
-CMD ["samtools"]
+CMD ["samtools"]/
```
Comment (dleehr) [inline samtools/Dockerfile.1.3]: Trailing `/`

**Comment by johnbradley** [comment]
Having trouble getting `dukegcb/spp` to build. Error: 
```
g++ -I/usr/share/R/include -DNDEBUG     -I./ -D_FASTMAP -DMAQ_LONGREADS	  -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c bed2vector.cpp -o bed2vector.o
In file included from /usr/include/c++/6/ext/hash_map:60:0,
                 from pc.h:4,
                 from bed2vector.cpp:1:
/usr/include/c++/6/backward/backward_warning.h:32:2: warning: #warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [-Wcpp]
 #warning \
  ^~~~~~~
bed2vector.cpp: In function ‘SEXPREC* read_bed_ends(SEXP)’:
bed2vector.cpp:118:55: error: template argument 3 is invalid
   hash_map<string, int, hash<string>,equal_to<string> > cind_map;
                                                       ^
```

**Comment by johnbradley** [comment]
@dleehr This is ready for review again.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
-GGR-Docker
+GGB-Dockerfiles
```
Comment (dleehr) [inline README.md]: GCB

File: README.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
 ==========
 
-Docker images for GGR
+Docker images for bioinformatic workflows that are built on [dockerhub](https://hub.docker.com/) under the [dukegcb organization](https://hub.docker.com/u/dukegcb/).
+
+## Source Repository Organization
+
+Each top level directory in the [GCB-Dockerfiles](https://github.com/Duke-GCB/GCB-Dockerfiles/) repository
+corresponds to a `dukegcb/<tool-name>` docker container. Under this directory will be a directory for each version of the software. Inside each subdirectory will be the Dockerfile. This approach was taken to work around a limitations where dockerhub will not let you specify build file (like the command line docker command does).
```
Comment (dleehr) [inline README.md]: What directory is the build context then? The one with the Dockerfile?

Comment (johnbradley) [inline README.md]: > What directory is the build context then? The one with the Dockerfile?
The directory containing the Dockerfile is the build context.

**Pull request merged**